com.ibm.wala.core/src/test11Subjects/java/privateInterfaceMethods/testDoubleReturn/ReturnDouble.java

```diff
@@ -0,0 +1,5 @@
+package privateInterfaceMethods.testDoubleReturn;
+
+import privateInterfaceMethods.PrivateInterface;
+
+public class ReturnDouble implements PrivateInterface {}
```

Comment:
I think we can delete this file; just having the `ReturnArray` test is good enough